Atomic/util.py

```diff
@@ -474,3 +500,11 @@ def validate_manifest(spec, img_rootfs=None, img_tar=None, keywords=""):
     if keywords:
         cmd += ['-k',keywords]
     return subp(cmd)
+
+def get_signature_path(atomic_conf):
```

Comment:
As atomic sign is for only creating a local signature, i have renamed this function to get_local_signature_path.  It is only used for figuring out the local path for storing signatures.  The points raised above do apply to the pull side of things.